includes/abstracts/abstract-wc-order.php

```diff
@@ -809,7 +808,7 @@ protected function get_items_key( $item ) {
 		} elseif ( is_a( $item, 'WC_Order_Item_Coupon' ) ) {
 			return 'coupon_lines';
 		} else {
-			return '';
+			return 'line_items';
```

Comment:
Added a new filter in the latest commit. You can never have too many filters, no?